common/build.gradle

```diff
@@ -23,6 +23,7 @@ test {
 }
 
 dependencies {
+    implementation(group: 'commons-io', name: 'commons-io', version: '2.12.0')
```

Comment:
common should have a minimal amount of dependencies. Let's not add this dependency just for reading the input stream

Reply:
Replaced call to `IOUtils.toString()` with a more verbose JDK-based implementation.